recipe/meta.yaml

```diff
@@ -92,14 +92,6 @@ outputs:
         - test -f "${PREFIX}/lib/libxgboost${SHLIB_EXT}"                  # [unix]
         - if not exist %LIBRARY_PREFIX%\mingw-w64\bin\xgboost.dll exit 1  # [win]
 
-  - name: _py-xgboost-mutex
-    version: 2.0
-    build:
-      string: {{ "cpu" if cuda_compiler == "None" else "gpu" }}_0
-    test:
-      commands:
-        - exit 0
-
   - name: py-xgboost
     script: >-
       {{ "install-py-xgboost.sh" if not win else "install-win-wrapper.bat" }}
```

Comment:
Currently we have the package `py-xgboost`, which builds and installs the Python package content